### PR TITLE
Update troubleshoot-endpoint-client.md

### LINKED ISSUE #1467: Broken Link (troubleshoot-endpoint-client.md)
In file:
sccm\protect\deploy-use\troubleshoot-endpoint-client.md

The following link is broken, cannot find alternative.

http://go.microsoft.com/fwlink/?LinkID=196174

### PR DESCRIPTION
**Pull request by AndrewJNet**
Original support link is broken, corrected link to Microsoft Support's "Contact Us" page since support has been moved under a central page. Resolves #1467

#MMS2019Docathon

### Summarize the change in the pull request title

Original support link is broken, corrected link to Microsoft Support's "Contact Us" page since support has been moved under a central page. Resolves #1467 

Fixes #1467

### CODE REVIEW

**Comment by PRMerger12** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @AndrewJNet for the contribution! I'll review after the MMS contest. 

#hold-off 
(Reviewers, please don't edit, I'll care of it.)

File: sccm/protect/deploy-use/troubleshoot-endpoint-client.md [hunk 1]
```diff
@@ -152,7 +152,7 @@ If you encounter problems with Windows Defender or Endpoint Protection, contact
 
 ### Step 7: Contact Support  
 
--   If the steps did not resolve the issue, contact support. For more information, see [Customer Support](http://go.microsoft.com/fwlink/?LinkID=196174) (http://go.microsoft.com/fwlink/?LinkID=196174).  
+-   If the steps did not resolve the issue, contact support. For more information, see [Customer Support](https://support.microsoft.com/en-us/contactus/).
```
Comment (aczechowski) [inline sccm/protect/deploy-use/troubleshoot-endpoint-client.md]: Remove the locale (`en-us/`) from the URL

**Comment by aczechowski** [comment]
Ping @AndrewJNet can you review and revise?

**Comment by PRMerger6** [comment]
@AndrewJNet : Thanks for your contribution! The author, @aczechowski, has been notified to review your proposed change.

**Comment by aczechowski** [comment]
Thanks @AndrewJNet 😄 
#sign-off 
*Once merged*, this should be live with the next daily publishing process. Right now that *should* be tomorrow (Wednesday, 22 May) around 11:00 Pacific time. No further action needed from you.

**Pull request merged**